docs/concepts/schedules.md

```diff
@@ -16,6 +16,13 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
 
 You can add a schedule to any flow [deployment](/concepts/deployments/). The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.
 
+There are four recommended possible ways to create a schedule for a deployment:
+
+- Use the Prefect UI
+- Use the `--schedule` flag with the CLI `deployment build` command
```

Comment:
Unfortunately we don't support a single `--schedule` flag, there are 3 separate flags, one for each schedule type; not sure of the best way to say that here.

Reply:
Good catch.  I'll change to "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command.